### PR TITLE
[ISSUE #3450]Add brpc example

### PR DESCRIPTION
**Pull request by MichaelDeSteven**
#3450

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-client/shenyu-client-brpc/src/main/java/org/apache/shenyu/client/brpc/common/annotation/ShenyuBrpcService.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+/**
+ * Shenyu brpc service annotation.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+@Inherited
+@RpcService
+@ShenyuBrpcClient
+public @interface ShenyuBrpcService {
```
Comment (dragon-zhang) [inline shenyu-client/shenyu-client-brpc/src/main/java/org/apache/shenyu/client/brpc/common/annotation/ShenyuBrpcService.java:36]: lost attributes.

Comment (MichaelDeSteven) [reply shenyu-client/shenyu-client-brpc/src/main/java/org/apache/shenyu/client/brpc/common/annotation/ShenyuBrpcService.java:36]: > lost attributes.

@ShenyuBrpcClient has enought attributes. @ShenyuBrpcService attributes can adding later when it is necessary.

Comment (dragon-zhang) [reply shenyu-client/shenyu-client-brpc/src/main/java/org/apache/shenyu/client/brpc/common/annotation/ShenyuBrpcService.java:36]: see https://github.com/apache/shenyu/issues/3805

Comment (MichaelDeSteven) [reply shenyu-client/shenyu-client-brpc/src/main/java/org/apache/shenyu/client/brpc/common/annotation/ShenyuBrpcService.java:36]: > see #3805

done.

**Review by dragon-zhang — APPROVED** [review]

**Pull request merged**